documentation-site/components/yard/type-definition.js

```diff
@@ -0,0 +1,385 @@
+/*
```

Comment:
Is this file from some other project? Should we have some tests?

Reply:
yeah this is from the extract-react-types package. I've vendored the code for now, but going to put up a pr to that package to help out

Reply:
related pr: https://github.com/atlassian/extract-react-types/pull/101